src/DynamoCore/Models/NodeModel.cs

```diff
@@ -610,6 +610,7 @@ public void SetNickNameFromAttribute()
         public event Action AstUpdated;
         public virtual void OnAstUpdated()
         {
+            MarkAsDirty(forceExecute:false);
```

Comment:
This will break things, if callers were to set `executionHints = ForceExecution`, then this clears it for them.